addons/web/static/src/views/basic_relational_model.js

```diff
@@ -865,7 +865,11 @@ export class StaticList extends DataPoint {
 
     async delete(recordId, operation = "DELETE") {
         const record = this.records.find((r) => r.id === recordId);
-        await this.__syncParent({ operation, ids: [record.__bm_handle__] });
+        const handle = record ? record.__bm_handle__ : undefined;
```

Comment:
maybe it's simpler to do:


```suggestion
        if (!record) {
            return;
         }
```

Reply:
Thanks for the review 🙂 Indeed we can do it a bit simpler, for the test, I'm not sure how can we do it as reproducing the exact same behavior from the user case seems difficult, I will try

Reply:
I've tried the suggestion but is giving failed runbot 🤔